src/features/pages/partners/deriv-prime/potential/data.tsx

```diff
@@ -0,0 +1,41 @@
+import BrokerageFirms from 'images/svg/deriv-prime/brokerage_firms.svg'
+import InvestmentFunds from 'images/svg/deriv-prime/investment_funds.svg'
+import AssetManagers from 'images/svg/deriv-prime/asset_managers.svg'
+import ProfessionalTraders from 'images/svg/deriv-prime/professional_traders.svg'
+import FinancialInstitutions from 'images/svg/deriv-prime/financial_institutions.svg'
+import CorporateClients from 'images/svg/deriv-prime/corporate_clients.svg'
```

Comment:
Performance team is already refactoring these kinds of import. Can it be fixed in this PR since this is a new set of images. Let's create an index file in `deriv-prime` directory and export as default all the images there.

Reply:
Hi @prince-deriv I have fixed this 🙏